Sprint-3/slideshow/index.html

```diff
@@ -3,12 +3,20 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Image Carousel</title>
+    <link rel="stylesheet" type="" href="style.css">
     <script defer src="slideshow.js"></script>
   </head>
   <body>
-    <img id="carousel-img" src="./assets/cute-cat-a.png" alt="cat-pic" />
+    <div id="imageDiv">
+    <img id="carousel-img" src="./assets/dog1.webp" alt="dog-pic" />
+    <p></p>
+    
+   <button type="button" id="autoBackeard-btn">AutoBackward</button>
```

Comment:
This doesn't look a consistent id value. Somehow it is used consistently in both HTML and JS code.

Reply:
> This doesn't look like a consistent id value. I am amazed that somehow it is used consistently in both HTML and JS code.

Dear CJ,
Thank you so much for reviewing my work.
I put "clearInterval(autoMoveInterval)" in lines 40 , 47 to avoid this happen.

Reply:
I was referring to `autoBackeard-btn`.